Content.Server/Lock/LockComponent.cs

```diff
@@ -14,3 +14,15 @@ public sealed class LockComponent : Component
         [ViewVariables(VVAccess.ReadWrite)] [DataField("lockingSound")] public SoundSpecifier LockSound { get; set; } = new SoundPathSpecifier("/Audio/Machines/door_lock_on.ogg");
     }
 }
+public sealed class LockToggleAttemptEvent : CancellableEntityEventArgs
+{
+    public bool Silent = false;
```

Comment:
You dont need to send this "silent" bool as AFAIK its only used in EntityStorage for playing audio.

Reply:
Nevermind, this is fine.